exporter/datadogexporter/examples/k8s-chart/configmap.yaml

```diff
@@ -66,6 +74,13 @@ data:
         send_batch_size: 100
         timeout: 10s
     service:
+      # This will enable otel collector to log in json format
+      telemetry:
+        logs:
+          encoding: "json"
+          initial_fields:
+            # service can be used to filter logs on DataDog UI
```

Comment:
Not sure what this means. But why do we need a filter here? Seems optional.

Reply:
It is not optional. `service` is Datadog special that needs to be include so that we can filter by service on Logs side. Collector isn't adding service, so we have to do that manually . I can try to reword it

Reply:
What I meant here is that the documentation isn't really clear. What are you actually trying to say? Before defining how something is used, we should first explain what it is or why it's here. For example:
```yaml
# Add the service field onto every log line. This can later be used to ...
```
Is this appropriate?